plutus-tx/src/PlutusTx/Builtins/HasBuiltin.hs

```diff
@@ -91,6 +92,10 @@ instance HasFromBuiltin a => HasFromBuiltin (BuiltinList a) where
     type FromBuiltin (BuiltinList a) = [FromBuiltin a]
     fromBuiltin (BuiltinList xs) = map fromBuiltin xs
 
+instance HasToBuiltin a => HasToBuiltin (Vector a) where
+    type ToBuiltin (Vector a) = BuiltinArray (ToBuiltin a)
+    toBuiltin = useToOpaque (BuiltinArray . map toBuiltin)
```

Comment:
This is causing a CI failure because `BultinArray` doesn't exist yet in this PR.  Can we just remove this from the PR?